From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Thu, 15 Oct 2015 17:44:48 +0200
Subject: Only set QVTK_FOUND if it's actually found

list(FIND ..) set's the variable to -1 if the string is not found, so
QVTK_FOUND was set regardless if it was installed or not. This uses the
shorter MATCHES syntax to test for the modules.
---
 cmake/Modules/FindQVTK.cmake | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/cmake/Modules/FindQVTK.cmake b/cmake/Modules/FindQVTK.cmake
index 1e9d683..9f48adf 100644
--- a/cmake/Modules/FindQVTK.cmake
+++ b/cmake/Modules/FindQVTK.cmake
@@ -21,12 +21,10 @@ if (${VTK_MAJOR_VERSION} VERSION_LESS "6.0")
     set (VTK_USE_QVTK ON)
   endif(NOT QVTK_FOUND)
 else (${VTK_MAJOR_VERSION} VERSION_LESS "6.0")
-  list (FIND VTK_MODULES_ENABLED vtkGUISupportQt GUI_SUPPORT_QT_FOUND)
-  list (FIND VTK_MODULES_ENABLED vtkRenderingQt RENDERING_QT_FOUND)
-  if (GUI_SUPPORT_QT_FOUND AND RENDERING_QT_FOUND)
+  if (";${VTK_MODULES_ENABLED};" MATCHES ";vtkCommonCore;" AND ";${VTK_MODULES_ENABLED};" MATCHES ";vtkRenderingQt;")
     set (VTK_USE_QVTK ON)
     set (QVTK_LIBRARY vtkRenderingQt vtkGUISupportQt)
-  else (GUI_SUPPORT_QT_FOUND AND RENDERING_QT_FOUND)
+  else ()
     unset(QVTK_FOUND)
-  endif (GUI_SUPPORT_QT_FOUND AND RENDERING_QT_FOUND)
+  endif ()
 endif (${VTK_MAJOR_VERSION} VERSION_LESS "6.0")
